js/index.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable no-unused-vars */
+const mobileMenuModulesCreator = (() => {
+  // eslint-disable-next-line quotes
+  const nav = document.querySelector('.my-nav');
+
+  let closeIcon;
+  let menuMobile;
+  let menuItems;
+
+  const menuItemsHTML = `
+    <li><a href="#portfolio">Portfolio</a></li>
+    <li><a href="#about">About</a></li>
+    <li><a href="#contact">Contact</a></li>
+  `;
+
+  function removeNavMobile() {
+    const navMobile = document.querySelector('.nav-mobile');
+    if (nav.contains(navMobile)) {
+      nav.removeChild(navMobile);
+    }
+  }
+
+  function createMenuItems() {
+    menuItems = document.createElement('ul');
+    menuItems.classList.add('menu-list');
+    menuItems.innerHTML = menuItemsHTML;
+    menuItems.querySelectorAll('a').forEach((item) => {
+      item.addEventListener('click', (event) => {
+        event.preventDefault();
+        const targetSection = document.querySelector(item.getAttribute('href'));
+        targetSection.scrollIntoView({
+          behavior: 'smooth',
+        });
+        mobileMenuModulesCreator.closeMenu();
+      });
+    });
+  }
+
+  function createCloseIcon() {
+    closeIcon = document.createElement('div');
+    closeIcon.classList.add('close-icon');
+    closeIcon.textContent = 'x';
```

Comment:
- [ ] It would be better to use x as a close icon and try to export the close icon from Figma  and using in your portfolio in order to match the Figma design 
- Yours 
<img width="81" alt="Screenshot 2023-05-15 at 23 15 20" src="https://github.com/ClaudiaRojasSoto/portfolio_1/assets/61790587/3d8aed92-1c28-4f5b-9996-46d88548216a">
- Figma 
<img width="95" alt="Screenshot 2023-05-15 at 23 15 30" src="https://github.com/ClaudiaRojasSoto/portfolio_1/assets/61790587/c99fa64a-8a8f-435e-bedd-113be38a9e3f">